### DISCUSSION TITLE
Drag and Drop to Switch output source

### DISCUSSION THREAD

**Discussion by ChocoboZ** [File-New-Project/EarTrumpet#1175] [Ideas]
Would it be possible to make the app switching a draggable feature instead of going through individual menus for each app?

Cheers,

**Comment by Raitoning** [comment]
I don't even know why this hasn't been implemented yet, it is a total must have ! I change my sound output every 10 minutes or so to swap between my headphones and my monitor depending on what I'm doing (especially Firefox since I use Discord on Firefox with headphones, but watch YouTube videos or other on the monitor). Drag'n'dropping would be a massive QoL improvement !

**Reply by ghost** [reply]
> I don't even know why this hasn't been implemented yet, it is a total must have ! I change my sound output every 10 minutes or so to swap between my headphones and my monitor depending on what I'm doing (especially Firefox since I use Discord on Firefox with headphones, but watch YouTube videos or other on the monitor). Drag'n'dropping would be a massive QoL improvement !

hey,
Can you add a feature Drag and drop application from audio device to another to change their playback device?
